docs/source/API/core/initialize_finalize/InitArguments.rst

```diff
@@ -0,0 +1,64 @@
+InitArguments
+=============
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Defined in ``<Kokkos_Core.hpp>`` header.
+
+.. warning:: Deprecated since 3.7
```

Comment:
we should tell user here to `use** ``Kokkos::InitializationSettings`` **instead**`